src/mpybit.py

```diff
@@ -0,0 +1,94 @@
+import time
+
+from addresses import addBMIfNotPresent, decodeAddress
+
+from bmconfigparser import BMConfigParser
+
+from helper_ackPayload import genAckPayload
+
+from helper_sql import sqlExecute
+
+from kivy.app import App
+from kivy.core.window import Window
+from kivy.uix.boxlayout import BoxLayout
+
+import queues
+
+import shutdown
+statusIconColor = 'red'
+
+
+class LoginScreen(BoxLayout):
+    """This will use for sending message to recipents from mobile client."""
+
+    def send(self):
+        """This used for sending message with subject and body."""
+        queues.apiAddressGeneratorReturnQueue.queue.clear()
```

Comment:
please maintain a function with 15-20 lines.
It will be easy to understand

Reply:
Hmm, the problem here is a that a lot of code is duplicate. It's better to do a separate refactoring for UI-independent functionality, and then have QT, kivy, curses, API and so on, to call these. If you separate it into functions now, then there is still refactoring to be done in the future anyway.

Or, partial refactoring can be done now. For example, a function for generating an address, and one for sending a message, that can be glued into any UI later. But it's more work.